web/src/store/publish.js

```diff
@@ -0,0 +1,28 @@
+import { publishRest } from '@/rest';
+
+export default {
+  namespaced: true,
+  state: {
+    versions: null,
+  },
+  mutations: {
+    setVersions(state, versions) {
+      state.versions = versions;
+    },
+  },
+  actions: {
+    async fetchPublishedVersions({ commit }, identifier) {
+      if (identifier) {
+        const response = await publishRest.get(`dandisets/${identifier}/versions/`)
+          .catch((error) => {
```

Comment:
I think using `try...catch` exclusively with `await` is cleaner in most cases, including here.